### PR TITLE
testkube-api: load files from configmap

### PR DESCRIPTION
**Pull request by dejanzele**
## Pull request description 

Load config files from configmap

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: charts/testkube-api/values.yaml [hunk 1]
```diff
@@ -62,18 +62,26 @@ cloud:
...
 slackToken: ""
+
 ## Slack secret to store slackToken, the key name should be SLACK_TOKEN
 slackSecret: ""
+
 ## Slack template for the events sent to Slack
 slackTemplate: ""
+
 ## Slack config for the events, tests, testsuites and channels
+slackConfig: ""
```
Comment (vsukhin) [inline charts/testkube-api/values.yaml:83]: should we add it to other values files?

File: charts/testkube-api/templates/deployment.yaml [hunk 2]
```diff
@@ -117,22 +115,12 @@ spec:
               {{- else }}
               value:  "mongo"
               {{- end }}
+            {{- if .Values.slackTemplate }}
```
Comment (vsukhin) [inline charts/testkube-api/templates/deployment.yaml:118]: is this for executors or slack template?

**Review by vsukhin — COMMENTED** [review]
a couple of questions

**Review by nicufk — APPROVED** [review]
Looks good, there is a typo that @vsukhin mentioned

**Pull request merged**